docs/appendices/release-notes/5.7.1.rst

```diff
@@ -46,6 +46,10 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Improve ingestion performance for partitioned tables where no primary keys are
+  defined by using an append only optimized ingestion strategy. The performance
+  improvement increases with the size of each shard.
+
```

Comment:
We have a fix in 5.6.5, this should go there as well?

Reply:
This is still a draft.